app/assets/src/components/PipelineSampleReads.jsx

```diff
@@ -473,6 +473,10 @@ class PipelineSampleReads extends React.Component {
       version_display = version_display + ', NR ' + this.pipelineRun.version.nr
     }
 
+    let run_date_display = !this.pipelineRun ? '' :
+                             !this.pipelineRun.created_at ? '' :
```

Comment:
could split into a few lines for readability?

Reply:
presenting a date right next to the version makes me think this is when the version was cut